gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -3323,6 +3327,22 @@ subroutine parse_fields(cfg, label, field_set, items, rc)
           if(present(items)) call items%push_back(item)
        enddo
        field_set%nfields = m
+!      check for duplicates
+       do i=1,field_set%nfields-1
+          do j=i+1,field_set%nfields
+             match_short_name = trim(field_set%fields(1,i)) == trim(field_set%fields(1,j))
```

Comment:
I'm not sure the `trim()` are needed here. @tclune would know for sure, but I think when you compare strings, like `x == y` then it just goes character by character to the right and as long as all characters in `x` are the same as `y`, it's equal. 

And I think if you have unequal strings, then the shorter is padded with spaces?

Reply:
Yep, appear to not need the trims